WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -85,8 +128,8 @@ private extension ProductCategoryListViewController {
     /// Displays the Sync Error Notice.
     ///
     func displaySyncingErrorNotice(retryToken: ProductCategoryListViewModel.RetryToken) {
-        let message = NSLocalizedString("Unable to load categories", comment: "Load Product Categories Action Failed")
-        let actionTitle = NSLocalizedString("Retry", comment: "Retry Action")
+        let message = Localization.synchErrorMessage
```

Comment:
Super nit: I see that we usually use `sync` as a short form. i.e. `syncError` seems to be the standard style in our codebase.